### PR TITLE
KTOR-1032 Migrate to Dokka 1.4.32

### PR DESCRIPTION
**Pull request by e5l**
Fixes [KTOR-1032](https://youtrack.jetbrains.com/issue/KTOR-1032)
Fixes [KTOR-2693](https://youtrack.jetbrains.com/issue/KTOR-2693)

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -222,74 +192,33 @@ if (project.hasProperty("enableCodeStyle")) {
...
 }
 
-if (project.hasProperty("enable-coverage")) {
-    apply(from = "gradle/jacoco.gradle")
-}
-
-// Disable binary compatibility check for JVM IR compiler output by default
-val jvm_ir_enabled: Boolean by extra
-val jvm_ir_api_check_enabled: Boolean by extra
+println("${rootDir.absolutePath}/docs")
```
Comment (rsinukov) [inline build.gradle.kts:217]: Please remove this `println` or add more information to the log string

File: build.gradle.kts [hunk 2]
```diff
@@ -222,74 +192,33 @@ if (project.hasProperty("enableCodeStyle")) {
...
-        if (it.hasProperty("compileKotlinJvm")) {
-            it.property("compileKotlinJvm")
-        } else null
-    }
-
-    tasks.create<org.jetbrains.dokka.gradle.DokkaTask>("dokkaWebsite") {
-        outputFormat = "kotlin-website"
-        outputDirectory = "${rootProject.projectDir}/apidoc"
+plugins {
+    id("org.jetbrains.dokka") version "1.4.32"
```
Comment (rsinukov) [inline build.gradle.kts:202]: Can it use `dokka_version` property from `gradle.properties` file?

**Review by rsinukov — APPROVED** [review]

File: build.gradle.kts (hunk 2 above)
Comment (e5l) [reply build.gradle.kts:202]: Nope, we can't do this in the plugins block

File: build.gradle.kts (hunk 1 above)
Comment (e5l) [reply build.gradle.kts:217]: Thanks, will remove

**Pull request merged**